### DISCUSSION TITLE
How to use the ContainerAppCommandInstruction

### DISCUSSION THREAD

**Discussion by lymberl** [dotnet/sdk-container-builds#547] [Q&A]
In the [documentaton](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container?pivots=dotnet-8-0#configure-container-commands) there is no example of how to use this. I have tried `<ItemGroup>
	<ContainerAppCommandInstruction Include="DefaultArgs" />
</ItemGroup>` but it doesnt seem to work for the following code in the csproj file.
`<ItemGroup Label="ContainerAppCommand Assignment">
	
	<ContainerAppCommand Include="echo" />
</ItemGroup>
<ItemGroup>
	<ContainerDefaultArgs Include="containersetupscript.sh" />
</ItemGroup>
<ItemGroup>
	<ContainerAppCommandInstruction Include="DefaultArgs" />
</ItemGroup>`
It doesnt work since it displays "dotnet dontetApp.dll containersetupscript.sh" instead of just "containersetupscript.sh". How to get rid of "_dotnet dontetApp.dll_" and just display "_containersetupscript.sh"_?

**Comment by Zygfryt** [comment]
It should work.

```
<PropertyGroup>
    ...
    <ContainerAppCommandInstruction>DefaultArgs</ContainerRepository>
</PropertyGroup>
<ItemGroup Label="ContainerAppCommand Assignment">
    <ContainerAppCommand Include="echo" />
    <ContainerDefaultArgs Include="containersetupscript.sh" />
</ItemGroup>
```
